### PR TITLE
DBZ-3822 GH Action that flags octocat commits

### PR DESCRIPTION
**Pull request by ani-sha**
https://issues.redhat.com/browse/DBZ-3822

### CODE REVIEW

**Comment by gunnarmorling** [comment]
If you wanted to test this, you could enable GH Actions for your fork of the debezium repo, merge this change there, and then send PRs against your fork until you're satisfied with the outcome.

**Comment by ani-sha** [comment]
@gunnarmorling Tried and tested, this should work after landing into `master`.

File: .github/workflows/octocat-commits-check.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            fi
+          done < SORTED_AUTHORS.txt
+      - name: Create comment
+        if: ${{ steps.octocat.outputs.OCTOCAT_COMMIT_FOUND == 'true' }}
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            Hi @${{ github.event.pull_request.user.login }}. Thank you for your valuable contribution.
+            Please author your commit using an email linked to your GitHub account.
```
Comment (gunnarmorling) [inline .github/workflows/octocat-commits-check.yml:31]:
```suggestion
            Please author your commit(s) using an email [linked to your GitHub account](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-graphs-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile).
```

**Review by gunnarmorling — APPROVED** [review]
Yeah! One question/suggestion inline. Thanks, @ani-sha!

File: .github/workflows/octocat-commits-check.yml (hunk 1 above)
Comment (ani-sha) [reply .github/workflows/octocat-commits-check.yml:31]: Yes it supports Markdown.

**Comment by gunnarmorling** [comment]
Very nice, thx!

@ani-sha, now with that comment plugin available, can we revive the idea of putting a comment a) to run the formatter if the formatting check failed, and b) to prefix each commit with a Jira issue number?

**Comment by ani-sha** [comment]
@gunnarmorling Sure. Will look into that.

**Pull request merged**